tensorflow/python/distribute/mirrored_strategy.py

```diff
@@ -681,6 +689,10 @@ def _num_replicas_in_sync(self):
   @property
   def worker_devices(self):
     return self._devices
+  
+  @property
+  def replication_mode(self):
```

Comment:
why do we need this property on the strategy?

Reply:
This is intended to address Priya's ask @ https://github.com/tensorflow/tensorflow/pull/38968#pullrequestreview-404701790